packages/wagtail/.storybook/__mocks__/eventEmitterMock.ts

```
@@ -0,0 +1,22 @@
+export class EventEmitter {
+  private events: Record<string, Function[]> = {}
+
+  on(event: string, listener: Function) {
```

Biome

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)